src/lightkurve/interact.py

```diff
@@ -1000,6 +1000,7 @@ def show_interact_widget(
     vmax=None,
     scale="log",
     cmap="Viridis256",
+    return_selection_mask=False,
```

Comment:
I think return_selected_mask is more apt (rather than selection) as it is actively selected when using interact.

Reply:
We could rename `return_selection_mask` to `return_selected_mask`.
I'll wait for any other suggested changes from review feedback, and update the codes in 1 batch.